index.js

```diff
@@ -86,3 +86,17 @@ if(score >=60 && score <= 64){
 if(score <= 59){
   console.log("Grade: Redo");
 }
+
+var table = require('markdown-table')
+  
+var t= table([
```

Comment:
Very cool. I'm glad there was a dep out there for this.

However, I don't see it in the `package.json`. How did you install it? We need to make sure it's added as a dependency in that file